pkg/crd/generator/generator.go

```diff
@@ -112,7 +109,7 @@ func (c *Generator) Do() error {
 		return fmt.Errorf("failed switching working dir: %v", err)
 	}
 
-	if err := b.AddDirRecursive("./pkg/apis"); err != nil {
+	if err := b.AddDirRecursive("./%s" + c.APIsPath); err != nil {
```

Comment:
`"path/filepath".Join` is probably safer here.

Reply:
that drops the leading `./` :(